x/oracle/keeper/historic_price_test.go

```diff
@@ -35,13 +36,15 @@ func (s *IntegrationTestSuite) TestSetHistoraclePricing() {
 	// check medians, num of available medians, and median standard deviation
 	medians := app.OracleKeeper.HistoricMedians(ctx, displayDenom, 3)
 	s.Require().Equal(len(medians), 3)
-	s.Require().Equal(medians[0], sdk.MustNewDecFromStr("1.2"))
-	s.Require().Equal(medians[1], sdk.MustNewDecFromStr("1.125"))
-	s.Require().Equal(medians[2], sdk.MustNewDecFromStr("1.1"))
+	s.Require().Equal(medians[0], *types.NewPrice(sdk.MustNewDecFromStr("1.2"), displayDenom, 17))
+	s.Require().Equal(medians[1], *types.NewPrice(sdk.MustNewDecFromStr("1.125"), displayDenom, 14))
+	s.Require().Equal(medians[2], *types.NewPrice(sdk.MustNewDecFromStr("1.1"), displayDenom, 11))
 
 	medianDeviation, err := app.OracleKeeper.HistoricMedianDeviation(ctx, displayDenom)
 	s.Require().NoError(err)
-	s.Require().Equal(medianDeviation, sdk.MustNewDecFromStr("0.111803398874989476"))
+	//s.Require().Equal(medianDeviation, []types.Price{*types.NewPrice(sdk.MustNewDecFromStr("0.111803398874989476"), displayDenom, 1)})
```

Comment:
Should this comment still be here?

Reply:
https://github.com/umee-network/umee/pull/1717/commits/79f914e3d9615ba81ce4cee9ada3e5841724a56e